app/ledger.js

```diff
@@ -761,8 +762,12 @@ var run = (delayTime) => {
 
   try {
     winners.forEach((winner) => {
-      var result = client.vote(winner)
+      var result
+      var siteSetting = appStore.getState().get('siteSettings').get(`https?://${winner}`)
```

Comment:
To avoid re-getting the appState multiple times, you can pull `appStore.getState().get('siteSettings')` out of the forEach loop